frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -764,8 +819,34 @@ export class TileViewPage extends React.Component {
     );
   }
 
+  renderGroupedItems(items, groupBy, renderTile) {
+    const groupedItems = filterByGroupByDropdown(items, groupBy);
+    return _.map(
+      groupedItems,
+      (value, key) =>
+        value.length > 0 && (
+          <>
+            <div className="co-catalog-page__group-title">
+              {key} ({_.size(value)})
+            </div>
+            <Gallery
+              gutter="sm"
+              className="catalog-tile-view-pf catalog-tile-view-pf-no-categories co-catalog-page__grid"
+            >
+              {_.map(value, (item) => (
+                <GalleryItem key={`gallery-${item.obj.metadata.uid}`}>
```

Comment:
This isn't consistent with the other gallery item.
It's also duplicate code that should be extracted and reused.

Reply:
Moved this part to `renderGalleryItem` and reusing it whereever required